frontend/assets/stylesheets/components/_bundle-header.scss

```diff
@@ -56,11 +67,34 @@
 /* Green Part of the Header */
 .bundle-header__main {
     background: #87be27;
-    padding-top: $gs-baseline * 2;
+    @include mq($from: phablet, $until: tablet) {
+        padding-top: $gs-baseline;
+    }
+    @include mq($from: tablet) {
+        padding-top: $gs-baseline * 2;
+    }
 }
 
 .bundle-header__main__content {
     @include bundle-column-wide();
+    @include mq($from: mobile, $until: mobile-wider) {
```

Comment:
Declaring `$from: mobile` may lead to anything that is at minimum 1px less than what we define as mobile (i.e. 320px) screwing with the CSS. For example, at `319px`:

![narrow-breakpoint](https://cloud.githubusercontent.com/assets/5131341/25906095/a8972036-359b-11e7-95da-8beb90db7b9f.png)


Is this something we care about? The percentage of viewports this small may be tiny so maybe not, but a possible solution is to drop the `$from` in these cases and just use the `$until`, so that it essentially just means `0px -> breakpoint_x`?